jujugui/templates/index.html.go

```diff
@@ -67,104 +67,61 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
       // the config file which is not yet available.
     </script>
   </head>
-
   <body>
-
-      <div style="display: none;">
-        {{/* Include the svg sprite image inside the body. */}}
-        {{.spriteContent}}
-      </div>
-
-      <div class="header-banner header-banner--left">
-        <div id="header-logo" class="header-banner__logo"></div>
-        <div id="header-breadcrumb"></div>
-        <div id="model-actions-container"></div>
-        <div id="provider-logo-container" class="header-banner__provider"></div>
-      </div>
-
-      <div class="header-banner header-banner--right">
-        <ul class="header-banner__list--right">
-          <li id="maas-server" style="display:none" class="header-banner__list-item">
-            <a href="" target="_blank" class="header-banner__link">MAAS UI</a>
-          </li>
-          <li id="header-search-container"
-              class="header-banner__list-item header-banner__list-item--no-padding"></li>
-          <li id="header-help" class="header-banner__list-item header-banner__list-item--no-padding"></li>
-          <li id="profile-link-container"
-              class="header-banner__list-item header-banner__list-item--no-padding"></li>
-        </ul>
-      </div>
-
-      <div id="zoom-container"></div>
-
-      <div id="full-screen-mask">
-        <div id="browser-warning" class="centered-column"
-            style="display:none;">
-          <svg class="svg-icon"
-            viewBox="0 0 90 35"
-            style="width:90px; height:35px;">
-            <use xlink:href="#juju-logo" />
-          </svg>
-          <div class="panel">
-            <div class="header">
-                  Your browser is not supported
-            </div>
-            <p>
-              If you continue to use Juju with your current browser your
-              experience may not be as good as we would like it to be.
-            </p>
-            <p>
-              Please use the latest version of
-              <a href="http://www.google.com/chrome">Chrome</a> to be fully
-              supported.
-            </p>
-            <form onsubmit="return continueWithCurrentBrowser();">
-              <input type="submit" class="button" value="Continue"/>
-            </form>
+    <div style="display: none;">
+      {{/* Include the svg sprite image inside the body. */}}
+      {{.spriteContent}}
+    </div>
+    <div id="app"></div>
+    <div id="full-screen-mask">
+      <div class="centered-column"
+        id="browser-warning"
```

Comment:
I'm not sure this browser warning is necessary any longer.

Reply:
Removed.